xbmc.py

```diff
@@ -308,17 +308,24 @@ def executeJSONRPC(jsonrpccommand):  # NOSONAR
     :rtype: str
 
     See https://codedocs.xyz/xbmc/xbmc/namespace_j_s_o_n_r_p_c.html
-
     """
+    from sakejsonrpc import JsonRpcApi
 
     json_data = json.loads(jsonrpccommand)
-    json_method = json_data["method"]
+    try:
+        # Implement some methods for real
+        return json.dumps(JsonRpcApi().handle(json_data))
+
+    except NotImplementedError:
+        # Fallback to stubs
+        pass
+
     json_responses = os.environ.get("KODI_STUB_RPC_RESPONSES")
     if not json_responses:
         raise ValueError(
             "Could not find JSON Response folder. Use the environment variable KODI_STUB_RPC_RESPONSES to set one")
 
-    path = "{}.json".format(os.path.join(os.path.join(json_responses, json_method.lower())))
+    path = "{}.json".format(os.path.join(json_responses, json_data["method"].lower()))
```

Comment:
@basrieter I have removed a duplicate `os.path.join` here. I don't think this was intentional?